graylog2-server/src/main/java/org/graylog2/rest/resources/system/logs/LoggersResource.java

```diff
@@ -152,7 +152,8 @@ protected void setLoggerLevel(final String loggerName, final Level level) {
         if(loggerName.equals(loggerConfig.getName())) {
             loggerConfig.setLevel(level);
         } else {
-            final LoggerConfig newLoggerConfig = new LoggerConfig(loggerName, level, loggerConfig.isAdditive());
+            boolean additive = LogManager.ROOT_LOGGER_NAME.equals(loggerConfig.getName()) || loggerConfig.isAdditive();
+            final LoggerConfig newLoggerConfig = new LoggerConfig(loggerName, level, additive);
```

Comment:
I'm curious. What's the use case for a new logger without additivity? We are not adding any appenders ourselves, or are we? So that would result in a logger without appenders. Is that even useful or could we just always set `additive` to true?

Reply:
In theory, if a logger is created in `log4j2.xml` for `org.graylog2` with `additivity=false` to use for example different appenders, this would ensure that child loggers for e..g. `org.graylog2.indexer` would inherit that config.

We don't use that and I don't know if anybody does, but I didn't want to break that compatibility.

Reply:
oh, now I see your point. `org.graylog2.indexer` won't output anything because this way it doesn't have any appenders.
we would either have to add the parent's appenders, but does that make much sense? maybe we should really just set it to `true`